core/arch/arm/plat-vexpress/platform_config.h

```diff
@@ -213,6 +213,9 @@
 #define GICD_OFFSET		0
 #define GICC_OFFSET		0x10000
 
+/* SDP memory: 31MByte reserved, see bios_qemu_tz_arm */
```

Comment:
ok, i see your point.
Yes, we could  locate the SDP memory inside qemu secure memory. 1MB for core,  12MB for TAs (+ pager page-store), 3 MB for SDP.